### PR TITLE
Rhui client package update

### PR DESCRIPTION
**Pull request by KarnVadaliya**

### CODE REVIEW

File: daisy_workflows/build-publish/rhui/rpms/rh-client-config.spec [hunk 1]
```diff
@@ -77,7 +79,8 @@ rm -rf $RPM_BUILD_ROOT
 %if 0%{?_gpg_keys:1}
     %config  /etc/pki/rpm-gpg/RPM-GPG-KEY-*
 %endif
+%attr(755,root,root) /etc/cron.daily/google-rhui-client-package-update
 
 %changelog
-* Sun Apr 17 2022 Liam Hopkins <liamh@google.com>
-- Created new client RPMs for RHUIv4 migration.
+* Fri Feb 10 2023 Google Compute Engine <images--associates@google.com>
```
Comment (zmarano) [inline daisy_workflows/build-publish/rhui/rpms/rh-client-config.spec:85]: use `gc-team@google.com` for these

File: daisy_workflows/build-publish/rhui/rpms/rh-client-config.spec [hunk 2]
```diff
@@ -77,7 +79,8 @@ rm -rf $RPM_BUILD_ROOT
 %if 0%{?_gpg_keys:1}
     %config  /etc/pki/rpm-gpg/RPM-GPG-KEY-*
 %endif
+%attr(755,root,root) /etc/cron.daily/google-rhui-client-package-update
```
Comment (zmarano) [inline daisy_workflows/build-publish/rhui/rpms/rh-client-config.spec:82]: Now that I found the [Fedora docs](https://fedoraproject.org/wiki/Packaging:CronFiles#Cron_Job_Files_Packaging) on this, it should be a config file with the noreplace annotation
%config(noreplace)

File: daisy_workflows/build-publish/rhui/rpms/rh-client-config.spec [hunk 3]
```diff
@@ -48,6 +48,8 @@ cp $RPM_BUILD_DIR/%{name}-%{version}/key.pem $RPM_BUILD_ROOT/etc/pki/rhui
 mkdir -p $RPM_BUILD_ROOT/%{_bindir}
 cp $RPM_BUILD_DIR/%{name}-%{version}/rhui-set-release $RPM_BUILD_ROOT/%{_bindir}
 
+# google-rhui-client-update cron job
+cp $RPM_BUILD_DIR/%{name}-%{version}/google-rhui-client-package-update $RPM_BUILD_ROOT/etc/cron.daily
```
Comment (zmarano) [inline daisy_workflows/build-publish/rhui/rpms/rh-client-config.spec:52]: You need to do the explicit directory creation before the copy
mkdir -p $RPM_BUILD_ROOT/etc/cron.daily

Comment (KarnVadaliya) [reply daisy_workflows/build-publish/rhui/rpms/rh-client-config.spec:54]: done

File: daisy_workflows/build-publish/rhui/rpms/rh-client-config.spec (hunk 2 above)
Comment (KarnVadaliya) [reply daisy_workflows/build-publish/rhui/rpms/rh-client-config.spec]: done

File: daisy_workflows/build-publish/rhui/rpms/rh-client-config.spec (hunk 1 above)
Comment (KarnVadaliya) [reply daisy_workflows/build-publish/rhui/rpms/rh-client-config.spec]: done

**Pull request merged**